ch.tigerjython.TigerJython2.yml

```diff
@@ -0,0 +1,49 @@
+id: ch.tigerjython.TigerJython2
+branch: '2.39'
```

Comment:
```suggestion
```

Reply:
How can I set the versionnumber for flatpak?

Reply:
It comes from the release tag in appdata file. You're missing an appdata file